app/src/main/java/com/hedvig/app/feature/marketpicker/MarketPickerFragment.kt

```diff
@@ -8,29 +8,32 @@ import com.hedvig.app.databinding.FragmentMarketPickerBinding
 import com.hedvig.app.feature.marketing.ui.MarketingActivity
 import com.hedvig.app.feature.marketing.ui.MarketingViewModel
 import com.hedvig.app.util.extensions.storeBoolean
+import com.hedvig.app.util.extensions.view.doOnLayout
 import com.hedvig.app.util.extensions.view.updateMargin
-import com.hedvig.app.util.extensions.viewBinding
 import dev.chrisbanes.insetter.doOnApplyWindowInsets
 import org.koin.android.ext.android.inject
 import org.koin.android.viewmodel.ext.android.sharedViewModel
 
 class MarketPickerFragment : Fragment(R.layout.fragment_market_picker) {
     private val viewModel: MarketPickerViewModel by sharedViewModel()
     private val marketingViewModel: MarketingViewModel by sharedViewModel()
-    private val binding by viewBinding(FragmentMarketPickerBinding::bind)
```

Comment:
What's going on here? 🤔

Reply:
Forgot to add this back after removing because of build errors, fixing